working/2364 - primary constructors/feature-specification.md

```diff
@@ -108,22 +114,25 @@ just after the class name specifies both a constructor declaration and a
 declaration of one instance variable for each formal parameter in said
 parameter list that has the _declaring_ modifier `var` or `final`.
 
-With this feature, the declaration of formal parameters as `final` will be
-a compile-time error. This ensures that `final int x` is unambiguously a
-declaring parameter. Developers who wish to maintain a style whereby formal
-parameters are never modified will have a lint to flag all such mutations.
+With this feature, all other declarations of formal parameters as `final`
+will be a compile-time error. This ensures that `final int x` is
+unambiguously a declaring parameter. Developers who wish to maintain a
+style whereby formal parameters are never modified will have a 
+[lint][parameter_assignments] to flag all such mutations.
+
+[parameter_assignments]: https://dart.dev/tools/linter-rules/parameter_assignmentshttps://dart.dev/tools/linter-rules/parameter_assignments
 
-Similarly, with this feature a formal parameter can not be declared with
-the syntax `var name`, it must have a type (`T name`) or the type must be
-omitted (`name`).
+Similarly, with this feature a regular (non-declaring) formal parameter can
+not be declared with the syntax `var name`, it must have a type (`T name`)
```

Comment:
Nit: "be declared with" -> "use".

It's a little confusing to have "declaring" and "declared" in the same sentence while referring to different concepts.